components/ActivityIndicator/src/MDCActivityIndicator.m

```diff
@@ -432,8 +432,13 @@ - (void)addStrokeRotationCycle {
 
   [CATransaction begin];
   {
+    __weak typeof(self) weakSelf = self;
     [CATransaction setCompletionBlock:^{
-      [self strokeRotationCycleFinishedFromState:MDCActivityIndicatorStateIndeterminate];
+      dispatch_after(dispatch_time(DISPATCH_TIME_NOW, (int64_t)(0.03 * NSEC_PER_SEC)),
+                     dispatch_get_main_queue(), ^{
+          MDCActivityIndicator *strongSelf = weakSelf;
+          [strongSelf strokeRotationCycleFinishedFromState:MDCActivityIndicatorStateIndeterminate];
+                     });
```

Comment:
I'm not sure I follow this change.
Let's catch up offline.

Reply:
I'd also like to see if we can determine why the client app has increased CPU but not the MDC Catalog.

Reply:
Please report back here after offline conversation to unblock.